sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -13,8 +13,15 @@ provider-id.setting-generator = "pluto provider-id"
 affected-services = ["kubernetes"]
 
 [metadata.settings.kubernetes.pod-infra-container-image]
-setting-generator = "schnauzer-v2 render --requires 'aws@v1' --requires 'kubernetes@v1(helpers=[pause-prefix])' --template '{{ pause-prefix settings.aws.region }}/eks/pause:3.1-eksbuild.1'"
-affected-services = ["kubernetes", "containerd"]
+affected-services = ["pod-infra-container-image"]
+
```

Comment:
The following applies to all migrations - 

https://github.com/bottlerocket-os/bottlerocket/commit/b46c719afa4998e77fc6617d011f06fd30cc2251 is a fairly clean example of adding a migration, which requires edits in several places:
* `Release.toml` (add these to a new 1.20.0 -> 1.21.0 section
* `sources/Cargo.toml`
* new crates under `sources/api/migration/migrations/v1.21.0` (copy an existing migration crate then edit its Cargo.toml and main.rs)